README.md

```diff
@@ -48,9 +48,21 @@ Ensure you are running Linux (kernel 3.8 or later) and have Docker
 
 CoreOS users see [here](https://github.com/fintanr/weave-gs/blob/master/coreos-simple/user-data) for an example of installing weave using cloud-config.
 
+For usage on OSX (e.g. with Docker Machine and/or Boot2Docker) we
+first need to make sure that the VM is running and configured before
+launching weave. For example, for a VM named "weave-1" on
+docker-machine:
```

Comment:
"...launching weave:". i.e. ditch the "For example..." sentence since it is quite obvious what the example is doing.